src/NotificationBar/NotificationBar.styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components';
+import { NotificationIcon, UpgradeIcon } from '../Icons';
+import { Props } from './NotificationBar';
+
+export const StyledNotificationBar = styled.div<Props>`
+  width: 100%;
+  min-height: 40px;
+  padding: 5px;
+  border: 1px solid;
+  display: flex;
+  justify-content: space-between;
+  ${p =>
+    p.variant === 'info' &&
+    css`
+      border-color: ${p.theme.colors.system.blue};
+      background: ${p.theme.colors.system.blueLighter};
+    `}
+
+  ${p =>
+    p.variant === 'alert' &&
+    css`
+      border-color: ${p.theme.colors.system.red};
+      background: ${p.theme.colors.system.redLighter};
+    `}
+`;
+
+export const StyledNotificationIcon = styled(NotificationIcon)`
+  padding-top: 10px;
+  padding-left: ${p => p.theme.spacing.base};
```

Comment:
```suggestion
  padding: 0 ${p => p.theme.spacing.base};
```

Or something similar to match closely to design